README_OACC.md

```diff
@@ -0,0 +1,89 @@
+# Building configuring for the OpenACC backend
+
+## CMake Basics
+
+Alpaka requires cmake version >= 3.15.
+
+In the root of the alpaka dir, run:
+```bash
+mkdir build
+cd build
+```
+
+## Configuring Using CMake
+
+In the build directory, invoke cmake to configure. Use the options below to
+enable only the OpenACC backend.
+
+```bash
+cmake .. \
+  -DALPAKA_ACC_ANY_BT_OACC_ENABLE=on \
+  -DBUILD_TESTING=on \
+  -Dalpaka_BUILD_EXAMPLES=on \
+```
+All other backends are disabled for faster compilation/testing and reduced
+environment requirements. The cmake package OpenACC is used to detect the
+required OpenACC flags for the compiler. Additional flags can be added, e.g:
+- gcc, target x86:
+  ```bash
+    -DCMAKE_CXX_FLAGS="-foffload=disable -fopenacc"
```

Comment:
only a note: after this PR is merged we should work on adding a CMake option e.g. `ALPAKA_OACC_TARGET` and cover the most used compilers.

Reply:
It actually use the `findPackage(OpenACC)`, -fopenacc is set automaticall (Will fix it alongside sqaushing commits). We could do something for the "-foffload=disable" for GCC.